src/Components/LogTable.css

```
@@ -15,6 +15,26 @@
 		position: relative;
 		word-wrap: break-word;
 		border-bottom: 1px solid var(--color-border);
+
+		button {
+			display: flex;
+            border: none !important;
+            background: none;
+            color: inherit;
+            border: none;
+            padding: 0;
```

I'd also add a margin to align button to the top of the row
```suggestion
            margin: 0;
            padding: 0;
```